### PR TITLE
style updates for presentation pages

### PR DESCRIPTION
**Pull request by Kataract**
with bonus speaker profile pic emphasis

### CODE REVIEW

File: src/pages/about/speakers-list.js [hunk 1]
```diff
@@ -24,8 +24,8 @@ export default class SponsorsPage extends React.Component {
             </div>
             <div className="speakers-list is-flex">
               {speakerList.map(({ node: speaker }) => (
-                  <Link to={`/speakers/${speaker.speaker_id}`} title={speaker.name}>
-                    <Img className="speaker-image-wrapper" fixed={speaker.photo.local.childImageSharp.fixed} />
+                  <Link to={`/speakers/${speaker.speaker_id}`}>
```
Comment (tylerdave) [inline src/pages/about/speakers-list.js]: I did miss adding the `alt` on he image but is there a problem w/ leaving the title on the link? W/ the title, hovering over the image shows the speaker name which I think is a nice addition. Alternatively I was originally trying to put the speaker name below the image but I couldn't figure out how to make that work right...

File: src/components/all.sass [hunk 2]
```diff
@@ -112,14 +112,37 @@ ul
...
+  align-items: center
+  flex-flow: row wrap
+  justify-content: space-evenly
+
+  a
+    display: block
+    margin: .5rem
+
+    .speaker-image-wrapper
+      border: 2px solid $pyohio-red
```
Comment (tylerdave) [inline src/components/all.sass]: I played w/ it a bit locally and I think I prefer this one w/ a 1px border. (I do like the 2px border on the larger version of the images on the talk/speaker profile pages.)

File: src/pages/about/speakers-list.js (hunk 1 above)
Comment (Kataract) [reply src/pages/about/speakers-list.js]: it's ill-advised to use `title` for a11y reasons: https://developer.paciellogroup.com/blog/2012/01/html5-accessibility-chops-title-attribute-use-and-abuse/

in retrospect I should have added speaker names as visible things and I can do that.  it should make all the placeholder images less weird looking.

File: src/components/all.sass (hunk 2 above)
Comment (Kataract) [reply src/components/all.sass]: ok, I can change that just for this spot.

File: src/pages/about/speakers-list.js (hunk 1 above)
Comment (tylerdave) [reply src/pages/about/speakers-list.js]: Good to know; thanks!

**Pull request merged**